react/features/toolbox/actions.web.js

```diff
@@ -83,10 +83,13 @@ export function hideToolbox(force: boolean = false): Function {
 
         dispatch(clearToolboxTimeout());
 
+        const focusSelector = '.toolbox-content-items:focus-within,.filmstrip:focus-within,.remotevideomenu:hover';
+
         if (!force
                 && (hovered
                     || state['features/invite'].calleeInfoVisible
-                    || state['features/chat'].isOpen)) {
+                    || state['features/chat'].isOpen
+                    || document.querySelector(focusSelector))) {
```

Comment:
What solution were you thinking about? It is intended that this checks the currently rendered DOM in realtime.

Reply:
Something like keeping focus state of the elements of the selector (e.g. toolbox-content-items, filmstrip etc) in the store and use it here. But I don't actually know how easy/viable is to do it...

Ideally we would fully manage state in react. I don't think is a very good idea mixing DOM query with react state. WDYT?

Reply:
In this case I think it would be less efficient to manage this in state because there can are a lot of element's involved that need to be tracked here. Basically all elements in Toolbox and Overflow Menus, Sidebar and more. So I believe using a DOM selector to just get elements with `:focus` or `:focus-within` is more efficient even if it doesn't feel right for a react app. I would argue Focus is a state of the rendered DOM and not necessarily one of the React Component.

Reply:
Fair enough.